source/index-management.rst

```diff
@@ -669,6 +669,13 @@ Common API Components
      * filter expression are included in the index.
      */
     partialFilterExpression: Document;
+
+    /**
+     * Optionally specifies a collation to use for the index. If not specified, the default collation
```

Comment:
I don't think we want to use the collation specified on the collection, do we? It seems like the user should be explicit, as with all other options.